pkg/cmd/version/version.go

```diff
@@ -56,19 +59,23 @@ func Command(p cli.Params) *cobra.Command {
 
 			cs, err := p.Clients()
 			if err == nil {
-				pipelineVersion, _ := version.GetPipelineVersion(cs)
+				pipelineVersion, _ := version.GetPipelineVersion(cs, namespace)
 				if pipelineVersion == "" {
-					pipelineVersion = "unknown"
+					pipelineVersion = "unknown, " +
+						"pipeline controller may be installed in another namespace please use tkn version -n {namespace}"
```

Comment:
Do we need this, as we are planning to no show version in case of unknown

Reply:
I see still it exist in the current code so in order to keep in sync with master added this change but if there is a plan to not to show `unknown` then i think we can remove while doing the PR related to removing `unknown` changes

WDYT?